tests/test_samplers.py

```diff
@@ -59,7 +59,7 @@ def setUp(self) -> None:
 
     def test_sample(self) -> None:
         # Generate negative sample
-        negative_batch = self.negative_sampler.sample(positive_batch=self.positive_batch)
+        negative_batch, _ = self.negative_sampler.sample(positive_batch=self.positive_batch)
```

Comment:
TL;DR: Here, this comment is unnecessary.

`_` is actually a variable name, although by code convention normally not used for "proper" variables. It however means that the second return value is bound to a variable and kept in memory until it gets ouf of scope. Thus, in particular when having pytorch code, it is usually preferrable to use 
```python
negative_batch = self.negative_sampler.sample(positive_batch=self.positive_batch)[0]
```

As said in the TL;DR, this is only a unittest so minimizing memory utilization shouldn't be of primary interest here 😅